WcaOnRails/app/assets/stylesheets/edit_events.scss

```diff
@@ -42,8 +42,13 @@
   }
 
   .panel-body {
-    select {
+    select[name="format"] {
       width: 100%;
     }
+
+    input[name="scrambleGroupCount"] {
```

Comment:
Don't we use hypens instead of camel case for such attributes?

Reply:
Actually, at the moment we are using camelcase on the edit events page. See https://github.com/jfly/worldcubeassociation.org/blob/0b4d5709421cd4e19559f9a8886ffc5b380b6f02/WcaOnRails/spec/features/competition_events_spec.rb#L45.

You're right that we probably use hyphens in other places, and it would be good to be consistent about this moving forward. For now, I think it's best to go with the thing that's "locally consistent".

Reply:
Ah, I wrote this with [this line in mind](https://github.com/jfly/worldcubeassociation.org/blob/74b630314eb0301f67562f563f4087555bbae8f8/WcaOnRails/app/javascript/edit-events/EditEvents.jsx#L205) ^^ This should be made consistent later on =)

Reply:
Ah, darn! I've changed that one to use camelcase =)